CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+##Contributing to Tasties project
+
+---
+
+Hello and welcome to Tasties project. 
+Please reading and following these guidelines will help us make the contribution process easy and effective for everyone involved.
+
+
+
+### Quicklinks
+
+---
+
+- [Pre-Requirements](#Pre-Requirements)
+- [Issues](#Issues)
+- [Pull Requests](#Pull Requests)
+- [Testing](#Testing)
+- [Code of Conduct](#Code of Conduct)
+- [Getting Help](#Getting Help)
+
+
+### Pre-Requirements
+
+---
+
+To contribution to our project please download and install the following:
+
+- Vagrant.
+- VirtualBox(or other VM that works with "Vagrant").
+
+After the installations typing 'vagrant up' and automatically the development environment will set up.
+
+
+### Issues
+
+---
+
+Issues should be used to report problems with the library, request a new feature, or to discuss potential changes before a PR is created.
+Please make sure that before the PR is created, the issue must be created and after the PR is created, the PR must be linked to the issue relevant to it.
+
+
+### Pull Requests
+
+---
+
+To create a pull request (PR) please following this instructions:
+
+1 Create an [issue](#Issues).
+2 Fork a repository and clone the repository.
+3 Create a new branch and new remote for the upstream repository.
+4 Make the changes on our local machine, add to the branch, commit the a massage and finally push the PR.
+5 Open a PR in our repository and follow this process: 
+	- Add a detailed description.
+	- Press on the Pull Requests.
+6 Link the issue to the PR when you open issues use there [words](#https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests).
+  Issues can also help the team know what every member of the team work on.
+
+
+###Pull Requests Standards
+
+---
+Our standards for pull request is:
+
+1 Make polite notes to each other without using bad language.
+2 The PRs should be a meaningful and concise title without issue numbers or lines code.
+3 Every PR should be linked to an issue.
+4 Each PR requires the approval of at least two team members.
```

Comment:
please fix newline spacing, looking at the rich diff

Reply:
@GalOved I think you missed this section :)
Please fix the spacing between the lines.